.github/workflows/vm-kernel.yaml

```diff
@@ -293,12 +309,26 @@ jobs:
           TAG: ${{ needs.get-tags.outputs.canonical }}
 
         run: |
-          result_image="${VM_KERNEL_IMAGE}:${TAG}"
+          result_image="${GHCR_REGISTRY}/${VM_KERNEL_IMAGE}:${TAG}"
 
           args=("-t" "${result_image}")
           for arch in ${ARCHS}; do
-            args+=("${VM_KERNEL_IMAGE}-${arch}:${TAG}")
+            args+=("${GHCR_REGISTRY}/${VM_KERNEL_IMAGE}-${arch}:${TAG}")
           done
           docker buildx imagetools create "${args[@]}"
 
           echo "image=${result_image}" >> $GITHUB_OUTPUT
+
+      - name: Login to Docker Hub
```

Comment:
Should this be kept up by the `Login to GHCR` step, like before?

Reply:
Done